oat-sa/tao-item-runner-qti-fe
# Feature/TR-765/load test taker response in audio PCI

Related to: https://oat-sa.atlassian.net/browse/TR-765

Exclude the `audioRecording` PCI from the list of disabled interactions in the `reviewRenderer`, so that the user can interact with it.
However, this will allow the user to record as well. This needs to be tackled in another PR, on the PCI itself.

Note: this PR does not ensure the audio recording is supplied to the PCI while reviewing it. This will be addressed by another PR on the relevant repository.

How to test:
- apply the PR (use the companion PR on taoQtiItem to install the package)
- take a test containing an audioRecording PCI (see the one linked to the ticket)
- one the test is complete, login to the backoffice, and go to the Results page
- review the response for the item containing the audio PCI.
- see if the controls are reachable.
